terratorch/tasks/wxc_downscaling_task.py

```diff
@@ -69,22 +71,25 @@ def configure_losses(self) -> None:
         Raises:
             ValueError: If *loss* is invalid.
         """
+        #TODO 'reduction' should be chosen using the config and 
+        # a similar class as IgnoreIndex should be defined for this class
+
         loss: str = self.hparams["loss"].lower()
         if loss == "mse":
-            self.criterion: nn.Module = nn.MSELoss(reduction="none")
+            self.criterion: nn.Module = nn.MSELoss(reduction="mean")
         elif loss == "mae":
-            self.criterion = nn.L1Loss(reduction="none")
+            self.criterion = nn.L1Loss(reduction="mean")
         elif loss == "rmse":
             # IMPORTANT! Root is done only after ignore index! Otherwise the mean taken is incorrect
-            self.criterion = RootLossWrapper(nn.MSELoss(reduction="none"), reduction=None)
+            self.criterion = RootLossWrapper(nn.MSELoss(reduction="none"), reduction="mean")
```

Comment:
@Joao-L-S-Almeida what was the reason to change this? would we need to expose this as parameter eventually? just making sure we don't hard code unexpected default behaviour...

Reply:
My bad, this choice is not necessary. I fixed it in the last commit.